### PR TITLE
WIP: Reorganize repository

### PR DESCRIPTION
**Pull request by stephanboersma**
Hi guys,

After our discussion Monday, I thought that I might be able to help out a little bit with cleaning up/simplifying the repository.

Then it should be easier for you to continue development. I did test locally but it is probably a good idea if one of you also run some tests :-)

You are of course free to accept or decline these updates.

**Updates**
1. Renamed CLI command from `react-bratus` to `bratus`. Personally, I think this gives a better experience in the terminal. Can easily be undone
2. _Parser_ classes and the `Server` class are now part of the `packages/cli` and are compiled together
3. Updated the `yarn run build`script. It will now compile the CLI and the React app, then move build files to the cli/bin, and finally link the cli locally for testing.
4. Remove `.DS_Store` and `lerna-debug.log` files + added them to `.gitignore`.
5. `/bin`and `/build` folders are also added to `.gitignore`. Note: Publishing a new version to npmjs would require the `bin` folder. Branching out to include the `bin`folder is required in the process.
6. Updated `README.md` to include a section about the development environment + reflect the command change from `react-bratus` to `bratus`

Let me know if you have any questions :-)

### CODE REVIEW

**Comment by thepiotrek** [comment]
Hey Stephan.

The changes sound good.
Thanks!

**Comment by stephanboersma** [comment]
@Evgenios95, If you want to merge then it should be approved :-)

**Comment by Evgenios95** [comment]
Hey Stephan! We know, I think we will merge it tomorrow. We just needed to
decide how we will approach it ☺️

File: packages/bratus-cli/src/api/Server.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import ASTParser from '@react-bratus/parser';
```
Comment (thepiotrek) [inline packages/bratus-cli/src/api/Server.ts:1]: This showed an error.
The Server.ts couldn't find ASTParser at this location.

Should this be updated to the new location?

`import ASTParser from '../parser/ASTParser';`

**Review by thepiotrek — COMMENTED** [review]
General comments:

* `tsconfig.json` in the bratus-cli directory is all red in VSCode, I don't understand what the problem is
* after running `yarn install` & `yarn build` (also tested `yarn run build`) bratus --help doesn't work. This is the response I get in the terminal: 

`lerna success run Ran npm script 'build' in 2 packages in 21.9s:
lerna success - @react-bratus/app
lerna success - @react-bratus/cli
$ cd packages/bratus-cli && yarn link
warning There's already a package called "@react-bratus/cli" registered. This command has had no effect. If this command was run in another folder with the same name, the other folder is still linked. Please run yarn unlink in the other folder if you want to register this folder.
✨  Done in 22.83s.
piot:react-bratus piotrek$ bratus --help
bash: bratus: command not found`

Maybe the problem is that I already have react-bratus installed?

File: packages/bratus-cli/src/api/Server.ts (hunk 1 above)
Comment (stephanboersma) [reply packages/bratus-cli/src/api/Server.ts:1]: Hi @thepiotrek,

Yes, you are correct.

**Comment by stephanboersma** [comment]
Guys,

There are some issues with this branch. I will close it and open a new one for review.

**Pull request closed without merging**